### ISSUE TITLE
Add support for TSS (titanium style sheets)

### ISSUE THREAD

**Issue by m1ga** [Glavin001/atom-beautify#122]
What would be the best way to include a custom language (TSS, a CSS style language for the app framework Titanium)?

~~~
"#btn_start":{
    top:15,
    right:15,
    width:100
}
"#table_word":{
    top: 75,
    bottom:75
}
~~~

Is there a way I can say that it would add \n after each , and a \t on each newline between { and } ?

**Comment by Glavin001** [comment]
@Prettydiff are we able to support this?

**Comment by prettydiff** [comment]
Assign this bug to me.  It will have to be an enhancement. It looks simple, but its not.  Notice that the property separators are commas instead of semicolons.  I can easily enhance support for this, but I scared about regression.  I just need to extra time to test it properly.

> Is there a way I can say that it would add \n after each , and a \t on each newline between { and }

I am not completely clear on this.  It sounds like "Allman" style braces instead of the more common "K&R" style braces.  I have an option for this on the Pretty Diff tool for JavaScript only.  If @m1ga could go to http://prettydiff.com/ and beautify some JavaScript with the "Allman (ANSI) style" option checked.  Just verify this is what you are looking for in CSS and I will ensure this feature is available to CSS.  Just scroll down the page to make the options visible on the Pretty Diff tool.

**Comment by Glavin001** [comment]
Created Issue for you, @prettydiff, at https://github.com/prettydiff/prettydiff/issues/57

**Comment by m1ga** [comment]
https://github.com/prettydiff/prettydiff/issues/57 is closed! Prettydiff now supports TSS files.

So prettydiff will be included in atom-beautify?

**Comment by Glavin001** [comment]
So the next issue is that Atom Beautify current uses the Grammar for the file as it is detected by Atom. However, I cannot find Atom language support package for TSS.
@m1ga please let me know if there is one, if not, I may need some more time to update for custom Grammars.

**Comment by m1ga** [comment]
I've started https://github.com/m1ga/language-tss a while ago. Not fully complete but it will apply when using tss files

**Comment by Glavin001** [comment]
Published to v0.24.0

Be sure to install https://github.com/m1ga/language-tss to allow Atom to detect TSS.

**Comment by m1ga** [comment]
:+1: awesome! Working like a charm
